lib/mapper.js

```diff
@@ -142,8 +142,9 @@ function formatRemarketingTag(msg) {
     bundleid: app.namespace,
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
     remarketing_only: 1,
+    usage_tracking_enabled: 1,
```

Comment:
On second thought, there's also this paragraph I just found:

"If you are only reporting certain custom events in your app for remarketing (and are not reporting app usage data on at least a daily basis to AdWords), don't send usage_tracking_enabled=1. Doing so will confuse AdWords into thinking that users who are not purchasing items should also be counted as users who are not opening your app at all."

It would seem this event is much less 'required' than it seems. Thoughts?